### PR TITLE
XIVY-14658 Fix homepage

### PR DESCRIPTION
**Pull request by jackra1n**
- Remove doubled preRenderView redirect
- Use the correct tasks model for open tasks

### CODE REVIEW

File: dev-workflow-ui/webContent/view/home.xhtml [hunk 1]
```diff
@@ -100,7 +89,7 @@
                   </p:link>
                 </p:column>
               </p:dataTable>
-              <h:panelGrid rendered="#{homepageIvyDevWfBean.tasksSize eq 0}">
```
Comment (ivy-cst) [inline dev-workflow-ui/webContent/view/home.xhtml:103]: Looks like you can also delete some stuff in the `homepageIvyDevWfBean`?

**Review by ivy-cst — APPROVED** [review]
Please merge now. Check later what is obsolete in the `homepageIvyDevWfBean`

File: dev-workflow-ui/webContent/view/home.xhtml (hunk 1 above)
Comment (jackra1n) [reply dev-workflow-ui/webContent/view/home.xhtml:103]: > Looks like you can also delete some stuff in the `homepageIvyDevWfBean`?

Yes. I think it's better to move the PersonalTasksModel into HomepageBean tho and completely delete the PersonalTasksBean

Comment (ivy-cst) [reply dev-workflow-ui/webContent/view/home.xhtml:103]: Yes that sounds like a good plan :+1:

**Pull request merged**